lib/widgets/priority_dropdown.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:flutter/material.dart';
+import 'package:geeruh/theme.dart';
+
+final List<String> priorityLevel = <String>['Low', 'Medium', 'High', 'Top'];
+final List<Image> priorityImages = <Image>[
+  Image.asset('images/PriorityLow.png'),
+  Image.asset('images/PriorityMedium.png'),
+  Image.asset('images/PriorityHigh.png'),
+  Image.asset('images/PriorityTop.png'),
+];
+
+String? currentValue = priorityLevel[0];
```

Comment:
Jeżeli inicjujemy wartość zmiennej, a potem nie przewidujemy tego, żeby była ona nullem, to można zrobić ja nienullowalną

Reply:
Trochę musztarda po obiedzie, ale nie definiujemy zmiennych globalnie, a lokalne stałe zawsze z "_" na początku. W tym momencie wszystkie 3 z tych wartości (priorityLevel, priorityImages, currentValue) są widoczne i modyfikowalne z zewnątrz